src/_pytest/config/__init__.py

```diff
@@ -634,7 +635,8 @@ def _rget_with_confmod(
     def _importconftest(
         self, conftestpath: Path, importmode: Union[str, ImportMode], rootpath: Path
     ) -> types.ModuleType:
-        existing = self.get_plugin(str(conftestpath))
+        conftestpath_str = str(conftestpath)
```

Comment:
I think we should create a function:

```python
def get_conftest_plugin_name(p: Path) -> str:
    return str(p)
```

and reuse that in both places here. This makes it standout more, plus we can document the issue (and reference the issue number) in the `get_conftest_plugin_name` docstring.

Reply:
Sorry, I am not sure I understood what you meant by "reuse that in both places here" since so for now I only renamed the variable to be more explicit `confestpath_str` -> `conftest_plugin_name`.

Maybe you were suggesting something like this?
```py
existing = self.get_plugin(get_conftest_plugin_name(conftestpath)
...
self.consider_conftest(mod, registration_name=get_conftest_plugin_name(conftestpath))
```

Both places are in the same function, so it feels slightly weird to call the helper function twice but maybe that is indeed what you wanted, anticipating further use of the `get_conftest_plugin_name` in other places of the code in the future?

Reply:
My intention is that having a function we make this issue more explicit, is easy to miss the significance of that tiny `str` call and why it was so important in the first place. Also the function let's us add more documentation/context.

But nevermind, given this is only local to the function I think the variable name is enough to convey the importance of that `str` call and it being reused in the  `self.consider_conftest`.